apph-back/src/main/java/com/viseo/apph/service/PhotoService.java

```diff
@@ -198,11 +204,36 @@ private String createFilterQuery(FilterDto[] filters) throws InvalidObjectExcept
         return query.toString();
     }
 
-    @Transactional
-    public PaginationResponse getUserFilteredPhotos(User user, FilterRequest filterRequest) throws InvalidObjectException {
-        String filterQuery = createFilterQuery(filterRequest.getFilters());
-        List<Photo> userPhotos = photoDao.getUserFilteredPhotos(user, filterQuery);
-        return getPaginationResponse(filterRequest.getPageSize(), filterRequest.getPage(), userPhotos);
+    private String createSortQuery(SortDto[] sortModel) {
+        if (sortModel.length == 0) {
+            return " ORDER BY p.id DESC";
+        }
+        StringBuilder query = new StringBuilder(" ORDER BY ");
+        switch (sortModel[0].getField()) {
+            case "title":
+                query.append("p.title");
+                break;
+            case "description":
+                query.append("p.description");
+                break;
+            case "creationDate":
+                query.append("p.creationDate");
+                break;
+            case "shootingDate":
+                query.append("p.shootingDate");
+                break;
+            case "size":
+                query.append("p.size");
+                break;
+            default:
+                query.append("p.id");
+        }
+        if ("asc".equals(sortModel[0].getSort())) {
```

Comment:
```suggestion
        if (sortModel[0].getSort().equals("asc")) {
```

ça change rien mais je trouve le code plus lisible comme ça.